<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialtool</class>
 <widget class="QDialog" name="serialtool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="cb_serialname">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="6">
    <widget class="QTextEdit" name="te_serialrecvmsg"/>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="cb_baund">
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="cb_databit">
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="cb_checkbit">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="cb_stopbit">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="cb_fluid">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CTS/RTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XON/XOFF</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pb_serialcon">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" rowspan="2">
    <widget class="QTextEdit" name="te_serialsendmsg"/>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pb_serialsend">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lb_serialtitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Serial Debug Tool</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
